crypto/bls12381/g2_test.go

```diff
@@ -265,8 +265,9 @@ func BenchmarkG2Add(t *testing.B) {
 }
 
 func BenchmarkG2Mul(t *testing.B) {
+	worstCaseScalar, _ := new(big.Int).SetString("ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff", 16)
```

Comment:
I didn't understand this change

Reply:
That was cherry picked from geth (https://github.com/ethereum/go-ethereum/pull/26447)